custom/samveg/case_importer/validators.py

```diff
@@ -95,7 +95,7 @@ def _validate_call_columns(cls, spreadsheet):
 
 class RequiredValueValidator(BaseRowOperation):
     @classmethod
-    def run(cls, row_num, raw_row, fields_to_update, import_context):
+    def run(cls, row_num, raw_row, fields_to_update, import_context, domain_name):
```

Comment:
`domain_name` is getting added a bunch of places where it isn't being used. What would you think of adding `kwargs` to `BaseRowOperation.__init__` and then having `UploadLimitValidator.__init__` expect to be passed a domain in kwargs?

I think this might mean shifting `BaseRowOperation` and its subclasses to be instantiated classes instead of using class methods for everything, so you'd create new validation objects in `samveg_case_upload_row_operations`, but I think that would be a fine change. What do you think?

Reply:
yeah, I think it'd be fine to keep `run` as a class method but use `kwargs` instead of the long param list

Reply:
Ok, but I ended up using instance method similar to what you had suggested earlier.

Reply:
yep, commented before i had finished reviewing, I think what you settled on is good